src/components/ServiceDetails/MediaGallery/index.js

```diff
@@ -10,11 +10,11 @@ import ArrowForwardIosIcon from "@material-ui/icons/ArrowForwardIos";
 import getVideoId from "get-video-id";
 
 import DefaultIconForVideo from "../../../assets/images/Play_1.png";
-import { HERO_IMG } from "../";
 
 import { useStyles } from "./styles";
 
 const mediaTypes = { IMAGE: "IMAGE", VIDEO: "VIDEO" };
+const MEDIA_GALLERY = "media_gallery";
```

Comment:
there could be multiple asset_types. 
So create an enum instead of just a constant. That will be better.

Reply:
there are but for media gallery we will be using only media gallery